### PR TITLE
Speed up bullet explosion, restrict shockwave radius

### PR DESCRIPTION
**Pull request by jeffsieu**
Makes bullet explosion look less like spraying out a fountain of sparks. The shockwave radius has also been amended to match the range of the dash impact.

The dash indicator has also been updated to fix translation problems due to incorrect rotation.

### CODE REVIEW

**Comment by owenl131** [comment]
LGTM

File: Scripts/Weapon/DashSkill.cs [hunk 1]
```diff
@@ -1,279 +1,288 @@
-using Godot;
```
Comment (daniellimws) [inline Scripts/Weapon/DashSkill.cs]: Is there CRLF change here? Why huge pile of code diff when there is no change?

Comment (jeffsieu) [reply Scripts/Weapon/DashSkill.cs]: hmm I'll have a look

Comment (jeffsieu) [reply Scripts/Weapon/DashSkill.cs]: ok fixed

**Comment by jeffsieu** [comment]
Feel free to squash.

**Comment by daniellimws** [comment]
ERROR: get_node: Node not found: Shockwave.

**Review by daniellimws — APPROVED** [review]

**Pull request merged**